src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -9,7 +9,9 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PlatformTarget>AnyCPU</PlatformTarget>
-	<DefineConstants>$(DefineConstants);PROJECTK</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetsWindows)' != 'true' ">
+    <DefineConstants>$(DefineConstants);MANAGED_SNI</DefineConstants>
```

Comment:
Why not use it everywhere?  What features are we missing on Windows that prevent this?